Automattic/wp-calypso
# Card: Move to calypso-ui

This is possibly the leafiest of leaf components, and it doesn't have any counterpart in `@wordpress/components`. Even `Ribbon` (#35768) relies on it (for the devdocs example).

#### Changes proposed in this Pull Request

Card: Move to calypso-ui

#### Testing instructions

```
npm run distclean && npm ci
npm start
```

Verify that the ribbon still works:

- http://calypso.localhost:3000/devdocs/design/card
- and pretty much all over Calypso
